nBTC/tests/redeem_workflow_tests.move

```diff
@@ -43,9 +46,73 @@ fun setup_redeem_test(
     (lc, ctr, redeem_id, dwallet_id, scenario, clock)
 }
 
+#[test_only]
+fun setup_redeem_test_for_confirmation(
```

Comment:
we have setup function! I tink this is share a lot of logic with this function I think